service/docs/source/geopm_pio.7.rst

```diff
@@ -73,11 +73,20 @@ Descriptions Of High Level Aliases
 
 ``BOARD_ENERGY``
     Total energy measured on the server's board. See :ref:`geopm_pio_cnl(7)
-    <geopm_pio_cnl.7:Requirements>` for signal availability requirements.
+    <geopm_pio_cnl.7:Requirements>` and :doc:`geopm_pio_msr(7)
+    <geopm_pio_msr.7>` for signal availability requirements.
```

Comment:
AFAICT based on the load order of the IOGroups specified in IOGroup.cpp, the MSRIOGroup's alias will always be preferred over the CNL one.  I think it's worth spelling this out here and also adding that:

On systems where the PLATFORM_ENERGY_STATUS MSR *AND* CNL are supported, the MSR is preferred.  If you need to access the CNL signal in that case, do not use this alias.